GPU/Common/TextureCacheCommon.cpp

```diff
@@ -906,9 +922,8 @@ bool TextureCacheCommon::MatchFramebuffer(
 
 	// If they match "exactly", it's non-CLUT and from the top left.
 	if (exactMatch) {
-		// TODO: Better checks for compatible strides here.
-		if (fb_stride != entry.bufw) {
-			WARN_LOG_ONCE(diffStrides1, G3D, "Found matching framebuffer at %08x with different strides %d != %d", fb_address, entry.bufw, (int)fb_stride);
+		if (fb_stride_in_bytes != tex_stride_in_bytes && TextureFormatMatchesBufferFormat(entry.format, fb_format)) {
```

Comment:
Isn't this impossible?  `exactMatch` will always be false if the byte stride doesn't match, now.

-[Unknown]

Reply:
Oops, yeah, this whole check can definitely be removed